app/controllers/api/proxy_rules_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Api::ProxyRulesController < Api::BaseController
+  before_action :authorize_rolling_update, :find_service, :find_proxy
+  before_action :find_proxy_rule, only: %i[edit update destroy]
+
+  activate_menu :serviceadmin, :integration, :mapping_rules
+
+  sublayout 'api/service'
+
+  def index
+    @proxy_rules = @proxy.proxy_rules.includes(:metric).ordered.paginate(page: params[:page])
```

Comment:
We could make it sortable like other index pages. See https://github.com/3scale/porta/pull/1008/commits/fd28f421bf2739bdd13872aad9fcc84334647cf7

Reply:
done!